src/test/kotlin/org/jetBrains/research/tasktracker/tracking/DefaultContentProviderTest.kt

```diff
@@ -0,0 +1,39 @@
+package org.jetBrains.research.tasktracker.tracking
+
+import org.jetBrains.research.tasktracker.config.MainTaskTrackerConfig.Companion.PLUGIN_NAME
+import org.jetBrains.research.tasktracker.models.Extension
+import org.jetBrains.research.tasktracker.tracking.task.Task
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import java.util.*
+
+class DefaultContentProviderTest {
+
+    private fun Extension.getExpectedFolderName(): String = this.name.lowercase(Locale.getDefault())
+
+    private fun Task.loadContent() = javaClass.classLoader
```

Comment:
Please, use the same structure in the resources folder: 
<img width="369" alt="Screenshot 2023-07-10 at 22 29 53" src="https://github.com/JetBrains-Research/tasktracker-3/assets/8993915/35dc8ea6-8bac-480d-a163-0712143bd6af">

In this case, you can rewrite this part in this way:
```kotlin
    private fun Task.loadContent() = DefaultContentProviderTest::class.java
        .getResource("${this.getFileName()}${getExtension().ext}")
        ?.readText()?.trimIndent() ?: error("Cannot find ${getFileName()} file")
```